tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+	"extends": "tslint:recommended",
+	"rules": {
+		"array-type": [true, "generic"],
+		"member-access": true,
+		"no-console": [false],
+		"no-duplicate-variable": true,
+		"object-literal-sort-keys": false,
+		"quotemark": [true, "single"],
+		"max-line-length": [
+			true,
+			{
+				"limit": 160,
+				"ignore-pattern": "^import | *export .*? {"
+			}
+		]
+	}
+}
```

Comment:
is this necessary?  Why don't just use eslint and tsconfig combination?

Reply:
Yeah We can just use them, but I am trying to follow the same Standard as other popular RC apps